examples/tutorials/core_api/2_checkpoints.py

```diff
@@ -55,7 +55,7 @@ def main(core_context, latest_checkpoint, trial_id, increment_by):
         time.sleep(.1)
         print("x is now", x)
         if batch % 10 == 9:
-            core_context.train.report_training_metrics(latest_batch=batch, metrics={"x": x})
+            core_context.train.report_training_metrics(latest_batch=batch+1, metrics={"x": x})
 
             # NEW: write checkpoints at regular intervals to limit lost progress in case of a crash
             # during training.
```

Comment:
you still need a `batch + 1` in save_state in this example (a few lines below this comment)

Reply:
yes, just realized that